Ports/.hosted_defs.sh

```diff
@@ -53,3 +53,8 @@ export PKG_CONFIG_SYSROOT_DIR="${SERENITY_BUILD_DIR}/Root"
 export PKG_CONFIG_LIBDIR="${PKG_CONFIG_SYSROOT_DIR}/usr/local/lib/pkgconfig"
 
 export SERENITY_INSTALL_ROOT="${SERENITY_BUILD_DIR}/Root"
+
+if [ -z "${SERENITY_PORT_DIRS:-}" ]; then
+    export SERENITY_PORT_DIRS=()
+fi
```

Comment:
How would this work with multiple directories in the env var exactly?

Reply:
this check is simply there to to see if `SERENITY_PORT_DIRS` was previously defined or not.

If it wasn't we simple set it to an empty list, the next line
```bash
SERENITY_PORT_DIRS+=("${SERENITY_SOURCE_DIR}/Ports")
```
converts the variable to a list, if it wasn't previously, and adds the contents to it.

Reply:
Sure - let's continue this discussion around the `README.md` addition!